Lib/fontbakery/profiles/adobefonts.py

```diff
@@ -390,6 +394,37 @@ def com_adobe_fonts_check_nameid_1_win_english(ttFont, has_name_table):
     return PASS, "Font contains a good Windows nameID 1 US-English record."
 
 
+@check(
+    id="com.adobe.fonts/check/unsupported_tables",
+    rationale="""
+        Adobe Fonts' font-processing pipeline does not support all kinds of tables
+        that can be included in OpenType font files.⏎
```

Comment:
Remove explicit U+23CE (?) following "files."

Reply:
This is intentional. Without it a line break is not created when the check is run. For context, see #3741.

Reply:
Ah! OK sorry, ignore 😸